EasyPost/Request.cs

```diff
@@ -16,11 +16,11 @@ public class Request
         private readonly Dictionary<string, object> _urlSegments;
         public string? RootElement { get; set; }
 
-        public Request(string iResource, Method method, Dictionary<string, object>? parameters = null)
```

Comment:
What prompted this change? Is there a downside to removing this?

Reply:
No downside. `.AddParameters()` is the preferred way to add parameters to a `Request` object, but I did add the ability to pass params to the constructor when refactoring this a few weeks ago. Noticed our library was about half-half on which it used, so decided to remove the confusion and make only one way to do it.